### PR TITLE
Doc: Update README.rst

### LINKED ISSUE #14: Add support for make.bat when building locally
The same way we have a Makefile, we should also provide a make.bat file that allows for building the cheat sheets locally. The two makefiles should behave equally and they should both have the same targets. @Revathyvenugopal162 please take care of this.

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
Add snapshot in readme and fix #14

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -59,8 +59,27 @@ This command uses latexmk to compile your TeX file into a PDF, and then convert
...
+   :width: 50%           
    
+
+`download pyfluent cheatsheet pdf <https://cheatsheets.docs.pyansys.com/pyfluent_cheat_sheet.pdf>`_
+
+.. image:: https://cheatsheets.docs.pyansys.com/pyfluent_cheat_sheet.png
+   :alt: pyfluent cheatsheet snapshot
+   :width: 50%
+
+`download pyaedt_API cheatsheet pdf <https://cheatsheets.docs.pyansys.com/pyaedt_API_cheat_sheet.pdf>`_
```
Comment (Revathyvenugopal162) [inline README.rst:75]:
```suggestion
`download pyaedt API cheatsheet pdf <https://cheatsheets.docs.pyansys.com/pyaedt_API_cheat_sheet.pdf>`_
```

File: README.rst [hunk 2]
```diff
@@ -59,8 +59,27 @@ This command uses latexmk to compile your TeX file into a PDF, and then convert
...
+   :alt: pyfluent cheatsheet snapshot
+   :width: 50%
+
+`download PyAEDT API cheatsheet pdf <https://cheatsheets.docs.pyansys.com/pyaedt_API_cheat_sheet.pdf>`_
+
+.. image:: https://cheatsheets.docs.pyansys.com/pyaedt_API_cheat_sheet.png
+   :alt: pyaedt API cheatsheet snapshot
+   :width: 50%
+
+`download PyEDB API cheatsheet pdf <https://cheatsheets.docs.pyansys.com/pyedb_API_cheat_sheet.pdf>`_
```
Comment (RobPasMue) [inline README.rst:81]: @Revathyvenugopal162 can you also add here the PyPrimeMesh cheat sheet?

File: make.bat [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+rem Note: If you are using make.bat file for building locally, change L2 of .tex files from 
+rem ``\usepackage{{./../static/style}}`` to ``\usepackage{{.\..\static\style}}``
+
+all: %CHEATSHEETS% 
+
+if "%1" == "pymapdl_cheat_sheet" goto pymapdl_cheat_sheet
+if "%1" == "pyfluent_cheat_sheet" goto pyfluent_cheat_sheet
+if "%1" == "pyaedt_API_cheat_sheet" goto pyaedt_API_cheat_sheet
+if "%1" == "pyedb_API_cheat_sheet" goto pyedb_API_cheat_sheet
```
Comment (RobPasMue) [inline make.bat:16]: PyPrimeMesh cheat sheet missing

**Review by RobPasMue — APPROVED** [review]
LGTM @Revathyvenugopal162 ... I completely forgot about this PR. Could you add my changes suggested and merge? I'll leave it approved!

**Pull request merged**